onedal/objective_function/tests/test_objective_function.py

```diff
@@ -0,0 +1,215 @@
+# ===============================================================================
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===============================================================================
+
+from daal4py.sklearn._utils import daal_check_version, sklearn_check_version
+from sklearn.metrics import log_loss
+from sklearn._loss.loss import HalfBinomialLoss
+from sklearn.linear_model import _linear_loss
+
+if daal_check_version((2023, "P", 100)):
```

Comment:
You already checking versioning and adding these test on build step. no need for these